lms/djangoapps/onboarding/models.py

```diff
@@ -901,3 +901,42 @@ class RegistrationType(models.Model):
         related_name='registration_type'
     )
     choice = models.SmallIntegerField(default=1, null=False)
+
+
+class Education(models.Model):
+    """
+    Model to store user education information from linkedin
+    """
+    linkedin_id = models.IntegerField(null=True, blank=True)
+    user = models.ForeignKey(User, related_name='linkedin_education',
+                             on_delete=models.CASCADE)
+    school_name = models.CharField(max_length=100, null=True, blank=True)
+    degree_name = models.CharField(max_length=100, null=True, blank=True)
+    start_month_year = models.DateField(null=True, blank=True)
+    end_month_year = models.DateField(null=True, blank=True)
+    description = models.CharField(max_length=500, blank=True, null=True)
+
+
+class Experience(models.Model):
+    """
+    Model to store user experience from linkedin
+    """
+    linkedin_id = models.IntegerField(null=True, blank=True)
+    user = models.ForeignKey(User, related_name='linkedin_experience',
+                             on_delete=models.CASCADE)
+    start_date = models.DateField(null=True, blank=True)
+    end_date = models.DateField(null=True, blank=True)
+    is_current = models.BooleanField(default=False)
+    title = models.CharField(max_length=50, null=True, blank=True)
+    company = models.CharField(max_length=100, null=True, blank=True)
+    summary = models.CharField(max_length=500, null=True, blank=True)
+
+
+class Skills(models.Model):
```

Comment:
It would be great if you make this name singular  just like other models (Education & Experience)

Reply:
done